eos_data_distribution/soma_subscription_fetcher.py

```diff
@@ -57,12 +57,18 @@ def __init__(self):
         self._subproducers = {}
 
     def _on_interest(self, o, prefix, interest, face, interestFilterId, filter):
-        # We handle one fo two paths:
+        # We handle one of two paths:
         #   /com.endlessm/subscriptions/soma/subscription/$sub_id/manifest.json/$chunk
         #   /com.endlessm/subscriptions/soma/shard/$shard_url/$chunk
 
         name = interest.getName()
-        chunkless_name = name.getPrefix(-1)
+
+        # XXX: Use more sophisticated parsing algorithm to strip non-chunk parts.
```

Comment:
You might as well split this code out into such a helper function now (even if the algorithm remains this simple), so that it can be easily reused in future without propagating more `XXX`s throughout the code.